public/js/components/AtomEdit/CustomEditors/RecipeFields/Ingredient.js

```diff
@@ -141,3 +155,26 @@ class Quantity extends React.Component {
     );
   }
 }
+
+class Units extends React.Component {
+  unitTypes = ['', 'cup', 'g', 'kg', 'oz', 'lb', 'bottle', 'floz', 'l', 'litre', 'ml', 'tsp', 'tbsp', 'dsp', 'bunch', 'cm', 'can', 'clove', 'dash', 'grating', 'handful', 'packet', 'piece', 'pinch', 'sheet', 'sprig', 'stick'];
+
+  static propTypes = {
+    fieldValue: PropTypes.string,
+    updateUnit: PropTypes.func.isRequired
+  };
+
+  updateIngredientUnit = (value) => {
+    this.props.updateUnit(value);
+  }
+
+  render () {
+    return (
+      <FormFieldSelectBox
```

Comment:
There are quite a lot of different unit types. Might it be possible to use something similar to the tag picker component to allow people to filter which unit they want?